.pre-commit-config.yaml

```diff
@@ -28,17 +28,17 @@ repos:
       - id: cargo-fmt
         name: cargo fmt
         entry: cargo fmt --
-        language: rust
+        language: system
```

Comment:
If I do not use `language: system`, when I run pre-commit locally, then I run `cargo test` for example everything recompile. It is like there is a Rust cache conflict.